2025-04-19 15:12:18,807 INFO    MainThread:36668 [wandb_init.py:setup_run_log_directory():647] Logging user logs to C:\Users\Usuario\ML_Proyect_6\deep\wandb\run-20250419_151218-ci35hdq7\logs\debug.log
2025-04-19 15:12:18,807 INFO    MainThread:36668 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to C:\Users\Usuario\ML_Proyect_6\deep\wandb\run-20250419_151218-ci35hdq7\logs\debug-internal.log
2025-04-19 15:12:18,807 INFO    MainThread:36668 [wandb_init.py:monkeypatch_ipython():599] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000025A9C85C3B0>
2025-04-19 15:12:18,807 INFO    MainThread:36668 [wandb_init.py:init():761] calling init triggers
2025-04-19 15:12:18,807 INFO    MainThread:36668 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'batch_size': 32, 'img_size': 224, 'epochs': 20, 'patience': 5, 'base_model': 'resnet18', 'learning_rate': 0.0005, 'weight_decay': 1e-05, 'unfreeze_layers': 2, '_wandb': {}}
2025-04-19 15:12:18,807 INFO    MainThread:36668 [wandb_init.py:init():784] starting backend
2025-04-19 15:12:18,807 INFO    MainThread:36668 [wandb_init.py:init():788] sending inform_init request
2025-04-19 15:12:18,823 INFO    MainThread:36668 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2025-04-19 15:12:18,824 INFO    MainThread:36668 [wandb_init.py:init():798] backend started and connected
2025-04-19 15:12:18,830 INFO    MainThread:36668 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-04-19 15:12:18,831 INFO    MainThread:36668 [wandb_run.py:_label_probe_notebook():1214] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-04-19 15:12:18,831 INFO    MainThread:36668 [wandb_init.py:init():891] updated telemetry
2025-04-19 15:12:18,835 INFO    MainThread:36668 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-04-19 15:12:20,858 INFO    MainThread:36668 [wandb_init.py:init():990] starting run threads in backend
2025-04-19 15:12:21,494 INFO    MainThread:36668 [wandb_run.py:_console_start():2375] atexit reg
2025-04-19 15:12:21,494 INFO    MainThread:36668 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-04-19 15:12:21,494 INFO    MainThread:36668 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-04-19 15:12:21,494 INFO    MainThread:36668 [wandb_run.py:_redirect():2315] Redirects installed.
2025-04-19 15:12:21,496 INFO    MainThread:36668 [wandb_init.py:init():1032] run started, returning control to user process
2025-04-19 15:22:04,704 INFO    MainThread:36668 [wandb_run.py:_finish():2112] finishing run 201905882-icai/deep-lab-models/ci35hdq7
2025-04-19 15:22:04,706 INFO    MainThread:36668 [jupyter.py:save_history():458] not saving jupyter history
2025-04-19 15:22:04,706 INFO    MainThread:36668 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-04-19 15:22:04,706 INFO    MainThread:36668 [wandb_init.py:_jupyter_teardown():581] cleaning up jupyter logic
2025-04-19 15:22:04,706 INFO    MainThread:36668 [wandb_run.py:_atexit_cleanup():2340] got exitcode: 0
2025-04-19 15:22:04,706 INFO    MainThread:36668 [wandb_run.py:_restore():2322] restore
2025-04-19 15:22:04,706 INFO    MainThread:36668 [wandb_run.py:_restore():2328] restore done
2025-04-19 15:22:08,428 INFO    MainThread:36668 [wandb_run.py:_footer_history_summary_info():3956] rendering history
2025-04-19 15:22:08,428 INFO    MainThread:36668 [wandb_run.py:_footer_history_summary_info():3988] rendering summary
2025-04-19 15:22:08,428 INFO    MainThread:36668 [wandb_run.py:_footer_sync_info():3917] logging synced files
